.circleci/config.yml

```diff
@@ -153,8 +153,11 @@ commands:
       - run:
           name: Deploying to k8s cluster for service << parameters.k8s_svc_name >>
           command: |
+            git clone https://github.com/binary-com/devops-ci-scripts
+            cd ./devops-ci-scripts/k8s-build_tools
+            chmod +x release.sh
```

Comment:
Nitpick: This shouldn't be here - should commit it with +x in the devops-ci-scripts
You can merge and fix later though.

Reply:
Done